bynder_local.module

```diff
@@ -126,14 +154,17 @@ function bynder_local_get_image_url(array $item) {
 /**
  * Implements hook_form_FORM_ID_alter().
  */
-function bynder_local_form_bynder_configuration_form_alter(&$form, \Drupal\Core\Form\FormStateInterface $form_state) {
+function bynder_local_form_bynder_configuration_form_alter(&$form, FormStateInterface $form_state) {
+  $field_map = Drupal::service('entity_field.manager')->getFieldMap();
 
-  $field_map = \Drupal::service('entity_field.manager')->getFieldMap();
+  $config = Drupal::config('bynder_local.settings');
+  $local_field = $config->get('local_image_field');
```

Comment:
Please, add a default value to avoid NULL.

Reply:
I've approached this in a different way; the value is now set via the module config on install, so we can always rely on it being set.